Orange/data/table.py

```diff
@@ -398,20 +398,20 @@ def match_type(x):
     def transform(self, domain):
         """
         Construct a table with a different domain.
-        
+
         The new table keeps the row ids and other information. If the table
         is a subclass of :obj:`Table`, the resulting table will be of the same
         type.
-        
+
         In a typical scenario, an existing table is augmented with a new
-        column by :: 
-        
```

Comment:
Fixup this commit into the commit that introduced the changes https://github.com/biolab/orange3/pull/2285/commits/e0488978bf6668fc7a113297037ca62df3dbb96b.

Set editor to Editor>General: Other: Strip trailing on Save = Modified; Always keep trailing on carrot = no.